foundation/www/conf/2018/index.html

```diff
@@ -21,7 +55,28 @@ <h3 class='mt-5'>Propose a Talk</h3>
 		<a href='mailto:foundation@erpnext.com' class='btn btn-dark btn-sm mt-4'>Send Your Proposal</a>
 	</div>
 </section>
-<section class='section-padding'>
+<section class='section-padding text-center'>
+	<div class='container'>
+		<h3 class='mt-5'>Sponsored By</h3>
```

Comment:
will be good to add a sub-title like, "A big thanks to our awesome sponsors who make this event happen"